### PR TITLE
Postgres GET and POST request

### PR DESCRIPTION
**Pull request by javsflores**
Created a server with a GET and POST request to postgress

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # productView
 SETUP:
 1: NPM install dependencies
-2: Seed database: npm run seed:db -> should result in 100 records inserted
-3: start server: npm start
+2: Seed database: npm run seed:db -> should result in 100 records inserted (MONGO DB related)
```
Comment (claire-melbourne) [inline README.md:4]: do we have to keep this info? the original repo still has it

File: README.md [hunk 2]
```diff
@@ -18,11 +18,13 @@ DISCLAIMER - Postgres must be installed in your system for the following steps.
...
 
     module.exports.initial = initial;
-6: npm run dataGenPG
+6: npm run dataGen
   Please wait til you see the following: finished data generation: X seconds
 7: npm run createPG
   Please wait til you see the following messages: Database successfully created!
   Table successfully created!
 8: npm run seedPG
   Please wait til you see the following: finished seeding Postgres database: X seconds
```
Comment (claire-melbourne) [inline README.md:27]: Nice signals to the developer to let them know whether things are going right or not!

File: server/pgClient.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+exports.getAll = (req, res) => {
+  let query = `SELECT * FROM records WHERE id=${req.params.id}`;
+  client.query(query, (error, results) => {
+    if (error) {
+      return console.log('error happened', error);
+    }
+    var result = [{
+      packaging: {
+        measurments: {
```
Comment (claire-melbourne) [inline server/pgClient.js:11]: measurements is missing and e. I like how you did this to get the camel casing back before it goes to front end!

File: server/pgClient.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      console.log('could not post', error)
+      return res.status(404).send();
+    }
+    console.log('posted')
+    return res.status(200).send();
+  })
+}
+
+// {
+//   "id": 19,
```
Comment (claire-melbourne) [inline server/pgClient.js:56]: probably delete this unless you need it for debugging (add a note that its for debugging?)

File: server/pgServer.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
+app.use(express.static(__dirname + '/../client/dist'));
+app.use('/*', (req,res, next) => {
+    res.header('Access-Control-Allow-Origin', '*');
+    next();
+});
+
+app.get('/api/productView/products/:id', client.getAll);
```
Comment (claire-melbourne) [inline server/pgServer.js]: This is cool! Going to need to look at express docs again to see how this works--> client.getAll is your call back?  Also though-- it seems like this is getting one product, not all right? or are you referring to getting all data for a product? Clarifying function name would make this clearer: getOne or getProduct?

**Review by claire-melbourne — APPROVED** [review]

File: server/pgServer.js (hunk 5 above)
Comment (javsflores) [reply server/pgServer.js]: nice @claire-melbourne , I think the "getProduct" name would be much clearer for the user. Will make that change ASAP.

File: server/pgClient.js (hunk 3 above)
Comment (javsflores) [reply server/pgClient.js:11]: @claire-melbourne yeah, the front-end is expecting it spelled like that, so I had to keep it the same. LOL. I'm trying not to mess with the front end at all right now, but I think eventually I will go and fix the spelling errors cause there are a few that I caught but couldn't change.

**Pull request merged**